subsys/fs/fuse_client/fuse_abi.h

```diff
@@ -0,0 +1,269 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
originally in Linux this file is licensed:
```
SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-2-Clause) 
```

looks like you are picking BSD-2 here, the question if this is something we can actually do, because here you are basically changing the license and dropping GPL. Need to check if we should keep the original license as is.

Reply:
Right.  I've changed the `fuse_abi.h` license so that the original identifier is preserved.